address/list.js

```diff
@@ -0,0 +1,96 @@
+var addr_default = [5,6,7,8];
+
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host1=window.location.origin+"/"+split_url[1];//+"/"+split_url[2];
+
+$(document).ready(function () {
+    $('[data-toggle="tooltip"]').tooltip();
+    $('#create').on('click', function () {
+        $('.show-create').html('Create Service Request !');
+        $('.modal-create').modal('show');
+    });
+
+    $('#add_addr').on('click', function(){
+        $('.show-addr').html('Filter Date');
+        $('.modal-map-addr').modal('show');
+    })
+
+});
+function checklist_tb(tstr, tstp) { //ditambahkan start dan stop date
+    datatable_list = $("#checklist").DataTable({
+      "ajax": apiUrl +'/cu?para=list_thermo&tstr='+tstr+'&tstp='+tstp,
+      "dom": '<"top"<"dt-filters"f>>rFt<"dt-bottom"<"dt-information"li><"dt-pagination"p>>',
+      buttons: [ 'colvis' ],
+      "columnDefs": [{
+          "targets": 'no-sort',
+          "serviceable": false,
+        },
+        {
+          "targets": addr_default,
+          "visible": false
+        }
+      ],
+        autoWidth : false,
+        iDisplayLength: 10,
+        bAutoWidth: false, 
+        fixedHeader : {
+          header: true, 
+        },
+        scrollY: "391px",
+        "scrollX": true,
+    //   "responsive": true,
+    //   "colReservice": {
+    //     realtime: false
+    //   },
+      "stateSave": false,
+      order: [[0, 'desc']],
+      createdRow: function (row, data, dataIndex) {
+        $(row).find('td:eq(0)').attr('style', 'cursor:pointer;');
+        $( row ).attr('data-id', data[12]);
+      },
+
+      initComplete: function () {
+        // Move Search To Panel Header
+        console.log($(this))
+        let _container = $(this).parents('.console-panel').find('.get_dt_search')
+        let _bottom_container = $(this).parents('.console-panel').find('.dt-bottom-container')
+        $("#checklist_wrapper .dataTables_filter input").appendTo(_container);
+        $("#checklist_wrapper  .dt-filters").css("display", "none");
+        // $(_container).find("input").attr('placeholder', 'Search From Table');
+        $(_container).find("input").css("display", "none");
+        $("#checklist_wrapper  .dt-bottom").appendTo(_bottom_container);
+        // dashboardFilters();
+        $(colvis.button()).insertAfter(_container);
+        $(colvis.button()).find('span:first').text("");
+        $(colvis.button()).find('span:first').attr('class', 'fas fa-eye');
+
+        set_tabel_session(); //function ada di search.js
+      }
+    });
+    var colvis = new $.fn.dataTable.ColVis(checklist, {
+      showAll: "Restore Defaults"
+    });
+}
+
+$(document).ready(function () {
+    if (sessionStorage.getItem(key_session_str) && sessionStorage.getItem(key_session_stp)) {
+        $('#reportrange').data('daterangepicker').setStartDate(moment(sessionStorage.getItem(key_session_str)).format('MM-DD-YYYY'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8393720)
```suggestion
        $("#reportrange").data('daterangepicker').setStartDate(moment(sessionStorage.getItem(key_session_str)).format('MM-DD-YYYY'));
```